garage/envs/mujoco/gather/gather_env.py

```diff
@@ -467,7 +466,7 @@ def log_diagnostics(self, paths, log_prefix='Gather', *args, **kwargs):
                 stripped_path[k] = v
             stripped_path['observations'] = \
                 stripped_path['observations'][
-                    :, :flat_dim(self.wrapped_env.observation_space)]
+                    :, :np.prod(self.wrapped_env.observation_space.low.shape)]
```

Comment:
ok you got me--i'm tired of reading np.prod(...). Can you put flat_dim back in?

Reply:
👍